arcsi/templates/base.html

```diff
@@ -28,18 +28,24 @@ <h1>
       <div class="navigation-block">
         <nav>
           <ul>
+            {% if current_user.has_role("admin") %}
             <li><a href="{{ url_for('router.add_show') }}">Add new show</a></li>
+            {% endif %}
             <li><a href="{{ url_for('router.list_shows') }}">Shows archive</a></li>
+            {% if current_user.has_role("admin") or current_user.has_role("host") %}
             <li><a href="{{ url_for('router.add_item') }}">Add new item</a></li>
+            {% endif %}
             <li><a href="{{ url_for('router.list_items') }}">Items archive</a></li>
           </ul>
         </nav> 
         <div>
           
           <div class="adminmenu">
             <a href="/doc" title="API documentation"><i class="fa fa-book"></i></a>
```

Comment:
do we need this `../doc` link for the show admins? I think it is only confusing, would put it in the if `has_role("admin")`